### ISSUE TITLE
American Indian/Native American/Indigenous Americans !!!FLAGS!!!

### ISSUE THREAD

**Issue by Thsar999** [flaghunters/Extra-Flags-for-4chan#289]
(See bottom of post for files)

Flags of a large amount of Native American reservations and what not for the United States and Canada. Also included are some tribes of Mexico, Central, and South America. I plan to do more but I wanted to push out this first batch before I go on a week and a half long trip and won't be able to do any more for that time frame.

I only included tribes that are recognized at the State or Federal levels, no unrecognized tribal entities are included and no tribes that don't have reservations or some sort of tribal land are included (My statement about reservations/tribal land does not apply to the tribes for Mexico, Central America, and South America. I have no idea about those, do as you please with them.)

I organized them according to state/province/territory/whatever and I think it should be done this way in the script as well because this would minimize people using Native American flags just to meme post (In other words, having to use a different state flag may discourage using them unless you're actually on reservation). This also means that some flags are included multiple times in different states as the reservation cross state lines, and in one case the reservation is actually on both sides of the Canadian-American border. Flags are also included in multiple states/countries when a tribe has reservations in multiple states or when multiple reservations/bands of the same tribe use the same flag (examples of this include the Mi'kmaq,the Arapaho, and the Iroquois)

Generally when there's a folder with a flag inside that has the same name of the folder (commonly with a folder inside where this same situation will repeat itself), this is my attempt at representing the layering of the flags. (ex: United States>New York>Iroquois>Oneida Nation>Oneida Nation of New York). If this is confusing feel free to reply to me and I'll try and sort up any confusion.

I used the Iroquois flag already in the script as a regional layer'd under different states with Iroquois reservations within them (those reservations regionals are then further layer'd under the Iroquois regional)

MasterAutismo's recent add on with county and city flags also included some Native Nation flags. I would recommend that his flags are used in place of some of mine because his are honestly better.

The replacements I recommend are: 

MasterAutismo's "Arapaho.png" in place of my "Arapaho Nation of Oklahoma.png" and my "Arapaho Tribe of the Wind River Reservation, Wyoming.png" (Note: both of these are the same flag)

MasterAutismo's "Cherokee.png" in place of my "Cherokee Nation, Oklahoma.png" (Note: This is the flag of the Cherokee Nation of Oklahoma, NOT the flag of all Cherokee)

MasterAutismo's "Choctaw.png" in place of my "Choctaw Nation, Oklahoma.png" (Note: This is the flag of the Choctaw Nation of Oklahoma, NOT the flag of all Choctaw)

MasterAutismo's "Navajo.png" in place of all three of my Navajo files under Arizona, New Mexico, and Utah

MasterAutismo's "Sioux.png" in place of my "Oglala Sioux Tribe of the Pine Ridge Reservation, South Dakota.png" (Note: This is the flag of the Oglala Sioux Tribe of the Pine Ridge Reservation in South Dakota NOT the flag of all Sioux)

[Indigenous flags of the Americas.zip](https://github.com/flaghunters/Extra-Flags-for-4chan/files/296705/Indigenous.flags.of.the.Americas.zip)

**Comment by karakalpak** [comment]
US and Canadian native flags should be accepted since their reservations are official and do have well-defined boundaries. Not to mention Iroquois has already been a flag for a long time.

With regards to Latin American flags, personally I think only the flags corresponding to existing native reservations should be added.

**Comment by MasterAutismo** [comment]
Fantastic and thorough job. I concur with karakalpak - the native tribe/peoples must control an officially demarcated piece of land in order to have their flag count. 

From the research I've done, the tribes of Canada and the US appear to control land while none of the Central/South American natives do (bar the Kuna of Panama with _comarcas_, but this is on a technicality as these have flags - the Kuna flag is merely tribal).

**Comment by karakalpak** [comment]
Just added the Native American flag, I'm probably going to sort all tribes into that folder (lads on /flag/ said it's a good idea). Not sure how I'm going to do it though

**Comment by Thsar999** [comment]
I'm pretty sure that's the flag of the Bureau of Indian Affairs that was used, which honestly doesn't seem right to me. Especially considering that many Native Americans disagree with current and past policies of the BIA.
The BIA was been involved in the suppression of Native American culture for an extremely long time, and more recently has even supported practical dictators within tribal governments (notable Dick Wilson in the 1970s) 
Also, the BIA doesn't hold any land, or power over the tribes.
I think the use of the BIA flag is incorrect, out of place, rude/disrespectful, and that most Native Americans wouldn't want to post under the BIA flag.

**Comment by karakalpak** [comment]
Right, will think about a better solution

**Comment by karakalpak** [comment]
Following the discussion on 
https://cuckchan.org/int/thread/61098265/#61122840
the status quo will be kept, will implement the flags at a later date

**Comment by Thsar999** [comment]
So, after finding more recent sources, I've find that some of the tribes should not be included due to no longer having reservations (Some were unrecognized at the federal level due to not meeting requirements, others closed their reservations due to lack of maintenance, etc.) and some of them I just seriously fucked up on and they never actually had reservations, and that's my mistake and I apologize.

**Anyways, here's the complete list of tribes that shouldn't be included:**

St. Francis-Sokoki band of the Missisquoi Abenaki, Vermont

Powhatan-Renape Nation, New Jersey

Houma Tribe, Louisiana

Mackinac Bands of Chippewa and Ottawa Indians, Michigan

Also the Chinook Nation of Washington can really only be included if the Confederated Tribes of the Grand Ronde Community of Oregon is also included, which they are a member of, along with a band of Iroquois that is. I didn't include a flag for this reservation, but they do have a flag that I will include next time I upload a bunch of Native American flags.

Basically the Chinook Nation had their federal status revoked, but not their state status, so they're still in that reservation. There's also a group of Iroquois on the reservation that aren't a member of the Iroquois confederacy. So, handle this entire situation however you want....I have no idea how I would honestly. It's a cluster fuck.

**Comment by Thsar999** [comment]
>tfw this hasn't been added yet
I was working on more, but I'm feeling pretty uninspired tbh

**Comment by MasterAutismo** [comment]
I'm looking at this issue. If the OP still wants to work on it, I'll add the flags. It's a fairly complicated issue due to sorting issues, but we can likely discuss it on /flag/ or on here.

**Comment by Thsar999** [comment]
I would like to keep working on it, but I'm afraid my time is much more limited now vs. back then.
I can upload the rest of what I have if you're interested, but if this is something people want done it's best for someone else to take over as I really don't have the time anymore because of work.
I feel bad about it tbh, but it is what it is.

**Comment by MasterAutismo** [comment]
Thought I responded to this, I guess I neglected. This is going to be the next major discussion on /flag/ and we'll decide whether or not to add these flags since this issue has been lurking for over a year.

**Comment by MasterAutismo** [comment]
The primary issue with this is that I'm not sure how to structure the reservations. Rather than make a discussion, I'll be creating a poll on whether to categorize these under their own independent folders within the country they exist in, OR whether to include them under regional jurisdictions (Navajo Nation in Arizona, Utah, Colorado, and New Mexico). 

Use of the BIA flag absolutely needs to be done away with, since the reservations ARE semi-autonomous, yet still dependent nations of the United States and other countries.

**Comment by MasterAutismo** [comment]
Actually, I'll be going with the latter option I listed above. Expect this issue to be closed within a week, likely much sooner.

**Comment by MasterAutismo** [comment]
Just dropping a quick note here to say I appreciate your hard work, Thsar999. I've finally added the US/Canada tribal flags to the best of my ability and the comarcas in Panama. It's been a long journey since you first submitted over a year ago, but they will be in very shortly.

If you still roam in /flag/, you're always free to drop by and say hello. In addition, if you have any comments, concerns, or questions - especially about this issue - feel free to ask in /flag/ or drop another comment here. Thank you for your contribution!
